flexget/plugins/modify/convert_magnet.py

```diff
@@ -33,8 +33,10 @@ def magnet_to_torrent(self, magnet_uri, destination_folder, timeout):
         import libtorrent
         params = libtorrent.parse_magnet_uri(magnet_uri)
         session = libtorrent.session()
-        # for some reason the info_hash needs to be bytes but it's a struct called sha1_hash
-        params['info_hash'] = bytes(params['info_hash'])
+        lt_version = [int(v) for v in libtorrent.version.split('.')]
+        if lt_version > [0,16,13,0]:
+            # for some reason the info_hash needs to be bytes but it's a struct called sha1_hash
+            params['info_hash'] = params['info_hash'].to_bytes()
```

Comment:
@cvium It's already in the return value of params, it's just that certain versions of libtorrent can't handle feeding its own data structure back in as 'info_hash', so this just does the cast. At least that's my understanding.